bin/scrape_software_versions.py

```diff
@@ -11,16 +11,17 @@
     'Bam2fasta': ['v_bam2fasta.txt', r"bam2fasta version (\S+)"],
     'fastp': ['v_fastp.txt', r"fastp (\S+)"],
     'Samtools': ['v_samtools.txt', r"samtools (\S+)"],
-    'SKA': ['v_ska.txt', r"Version (\S+)"],
+    'SKA': ['v_ska.txt', r"Version: (\S+)"],
     'htslib': ['v_samtools.txt', r"htslib (\S+)"],
     'Sourmash': ['v_sourmash.txt', r"sourmash (\S+)"],
     'SortMeRNA': ['v_sortmerna.txt', r"SortMeRNA version (\S+),"],
 }
 results = OrderedDict()
 results['nf-core/kmermaid'] = '<span style="color:#999999;">N/A</span>'
 results['Nextflow'] = '<span style="color:#999999;">N/A</span>'
-results['bam2fasta'] = '<span style="color:#999999;">N/A</span>'
+results['Bam2fasta'] = '<span style="color:#999999;">N/A</span>'
```

Comment:
```suggestion
results['bam2fasta'] = '<span style="color:#999999;">N/A</span>'
```

below softwares are not capitalized this way